README.md

```diff
@@ -258,10 +257,6 @@ To enable this feature, the `UMF_BUILD_SHARED_LIBRARY` option needs to be turned
 
 TODO: Add a description
```

Comment:
could you pls add some description here?

and also mention that it is `(part of libumf)` now